### PR TITLE
Issue 366/update spa template policies

### PR DESCRIPTION
**Pull request by godanny86**
Updates the Template policies for projects generated using Angular or React SPA front-end builds.

## Description
* Creates a Responsive Grid (Layout Container) policy used for Angular/React templates that includes only SPA-enabled components
* Cleans up auto-generated content under `/content/__appId__/language-masters/en` and `/content/__appId__/us/en` to remove non-spa components and references to ContextHub (since it is not supported with SPA-Editor)
* Removes the `/$confFolder/settings/wcm/templates/page-content` template using the `archetype-post-generate.groovy` script for SPA-enabled projects (since this template won't work inside a SPA project)
* Added a template thumbnail for `spa-app-template` and `spa-page-template`

## Related Issue

* Fixes Issue #366 (original issue filed)
* Fixes Issue #371  (possible duplicate issue)
* Potential fix for issue #372 

## Motivation and Context

SPA-enabled projects generated by the archetype should include only components that are SPA-ready.

## How Has This Been Tested?

I have generated **React** and **Angular** test projects and installed on the latest AEM as a Cloud Service Instance and AEM 6.5 + SP3. I have also generated a vanilla AEM project using the **General** build to ensure that non-SPA policies are not effected.

## Screenshots (if appropriate):

**Default Page Policy for SPA**: 
  image

**Templates generated for SPA projects**:
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by godanny86** [comment]
@ksurendra FYI I created this PR and I think it might address some of the issues you recently filed. It would be great if you could also check it out. Thanks!

File: src/main/archetype/ui.content/src/main/content/jcr_root/content/__appId__/language-masters/en/.content.xml [hunk 1]
```diff
@@ -26,22 +28,66 @@
...
                 <title
                     jcr:primaryType="nt:unstructured"
-                    jcr:title="Welcome to My Site!"
                     sling:resourceType="${appId}/components/title"/>
                 <container
                     jcr:primaryType="nt:unstructured"
                     sling:resourceType="${appId}/components/container"
                     layout="responsiveGrid">
-                    <text
+                    <teaser
```
Comment (CezCz) [inline src/main/archetype/ui.content/src/main/content/jcr_root/content/__appId__/language-masters/en/.content.xml:36]: Do we want to have these elements added too ?

Comment (godanny86) [reply src/main/archetype/ui.content/src/main/content/jcr_root/content/__appId__/language-masters/en/.content.xml:36]: @CezCz i'm not sure what you mean? The only spa (react, angular) components that are implemented is the text component and layout container AFAICT...

**Comment by ksurendra** [comment]
> @ksurendra FYI I created this PR and I think it might address some of the issues you recently filed. It would be great if you could also check it out. Thanks!

Looks good @godanny86 
- Merged to "master" 
- Rebuilt the Archetype
- Built a project with the local Archetype

Can Create a new page using "SPA Page" template - Fixes #372 
Can use the default page built by the archetype - Fixes #371

**Comment by godanny86** [comment]
hi @CezCz any update on this?

**Pull request merged**